python/psautohint/__main__.py

```diff
@@ -648,6 +647,15 @@ def get_options(args):
     )
     parsed_args = parser.parse_args(args)
 
+    log_level = logging.WARNING
```

Comment:
I think ERROR messages should be seen regardless of the verbose level. I tried to test it but ran into other unrelated issues. Leave the setup of ERROR as is for now until it can be how it's working.

Reply:
OK.